docs/medical-api/handling-data/schema/condition.mdx

```diff
@@ -4,48 +4,47 @@ description: "Schema documentation for the Condition table in the Metriport data
 
 **Primary Keys:**
 * condition_id
 
 **Foreign Keys:**
-* person_id
-* member_id
 * patient_id
-* encounter_id
 
-| Column | Data Type | Description | Terminology |
-| ------ | --------- | ----------- | ----------- |
-| condition_id | string | Unique identifier for the condition | - |
-| person_id | string | Unique identifier for the person | - |
-| member_id | string | Unique identifier for the member | - |
-| patient_id | string | Unique identifier for the patient | - |
-| encounter_id | string | Unique identifier for the encounter | - |
-| condition_type | string | Type of condition (e.g., problem, symptom, diagnosis) | - |
-| condition_code | string | Standardized code for the condition | ICD-9-CM, ICD-10-CM, SNOMED CT |
-| condition_code_type | string | Type of coding system used | icd-9-cm, icd-10-cm, snomed-ct |
-| condition_code_system | string | Coding system identifier | - |
-| condition_code_version | string | Version of the coding system | - |
-| condition_code_display | string | Human-readable description of the condition | - |
-| condition_rank | integer | Rank/priority of the condition (1 = primary, 2 = secondary, etc.) | - |
-| recorded_date | date | Date when the condition was recorded | - |
-| onset_date | date | Date when the condition began | - |
-| abatement_date | date | Date when the condition resolved | - |
-| clinical_status | string | Clinical status of the condition | active, inactive, resolved |
-| verification_status | string | Verification status of the condition | confirmed, provisional, differential |
-| severity | string | Severity of the condition | mild, moderate, severe |
-| category | string | Category or classification of the condition | - |
-| body_site | string | Anatomical location of the condition | - |
-| laterality | string | Side of the body affected | left, right, bilateral |
-| encounter_class | string | Class of the associated encounter | inpatient, outpatient, emergency |
-| facility_type | string | Type of healthcare facility | hospital, clinic, urgent_care |
-| data_source | string | Source of the condition data | claims, clinical, ehr |
-| source_code | string | Original code from the source system | - |
-| normalized_code | string | Standardized/normalized code | - |
-| source_system | string | System that provided the condition data | - |
-| source_table | string | Table in the source system | - |
-| source_id | string | Identifier from the source system | - |
-| ingestion_date | timestamp | Date when the data was ingested | - |
+| Column | Data Type | Description |
+| ------ | --------- | ----------- |
+| condition_id | string | Unique identifier for the condition |
+| patient_id | string | Unique identifier for the patient |
+| recorded_date | date | Date when the condition was recorded |
+| onset_date | date | Date when the condition began |
+| end_date | date | Date when the condition ended |
```

Comment:
If we're renaming this to end_date, shouldn't we also rename onset_date to start_date to keep it consistent?

Reply:
I said "...shouldn't we also rename `onset_date` to `start_date`..."

so yeah, they're opposite and end date is the opposite of start date, more clearly, I mean